extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -161,3 +175,43 @@ class="jetpack-podcast-player__episode-link"
 
 	return ob_get_clean();
 }
+
+/**
+ * Given the color name, bock attributes and the CSS property,
+ * the function will return an array with the `class` and `style`
+ * HTML attributes to be used straight in the markup.
+ *
+ * @example
+ * $color = get_colors( 'secondary', $attributes, 'border-color'
+ *  => array( 'class' => 'has-secondary', 'style' => 'border-color: #333' )
+ *
+ * @param string $name     Color attribute name, for instance `primary`, `secondary`, ...
+ * @param array  $attrs     Block attributes.
+ * @param string $property Color CSS property, fo instance `color`, `background-color`, ...
+ * @return array           Colors array.
+ */
+function get_colors( $name, $attrs, $property ) {
+	$attr_color  = "{$name}Color";
+	$attr_custom = 'custom' . ucfirst( $attr_color );
+
+	$color        = isset( $attrs[ $attr_color ] ) ? $attrs[ $attr_color ] : null;
+	$custom_color = isset( $attrs[ $attr_custom ] ) ? $attrs[ $attr_custom ] : null;
+
+	$colors = array(
+		'class' => '',
+		'style' => '',
+	);
+
+	if ( $color || $custom_color ) {
+		$colors['class'] .= " has-{$name}";
+
+		if ( $color ) {
+			$colors['class'] .= ' ' . "has-{$property}-{$color}";
+		} elseif ( $custom_color ) {
+			$colors['style'] .= "{$property}: {$custom_color};";
+		}
+	}
+
+	$colors['class'] = trim( $colors['class'] );
```

Comment:
Not sure we need to trim with above changes. If we do, this could work as well: `return array_map( 'trim', $color );`.

Reply:
> Not sure we need to trim with above changes

it's coating the CSS classes, so before to return these values I remove the spaces in extremes.

> If we do, this could work as well: return array_map( 'trim', $color );.
👍